src/main/resources/divolte.js

```diff
@@ -183,7 +183,7 @@ var AUTO_PAGE_VIEW_EVENT = true;
         // Assumes cookie name and value are sensible. (For our use they are.)
         // Note: No domain means these are always first-party cookies.
         var cookieString = name + '=' + value + "; path=/; expires=" + expiry.toUTCString() + "; max-age=" + maxAgeSeconds;
-        if (domain) {
+        if (domain && domain.length) {
```

Comment:
The compiled JavaScript is only different when using `''` instead of `""` for the config value as you mentioned in #121. I still think this is a valid fix for that particular use-case, but it might be more prudent to make the JavaScript more closely resemble the JavaScript produced by the Closure Compiler.